### DISCUSSION TITLE
[Vote ended on 2023-01-26] Update Ansible PyPI description

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#186] [Votes done]
There has been a proposal for some time to update the Ansible PyPI description (https://github.com/ansible-community/community-topics/issues/173): https://github.com/ansible-community/antsibull/pull/474

Since everyone seems to like the parts already changed, we want to vote about merging **the current version**. There was some more discussion about adjusting other parts, these can happen in (a) follow-up PR(s) later on.

Please vote +1 for merging https://github.com/ansible-community/antsibull/pull/474 as-is (up to typos, if someone finds some), or -1 for not merging it yet.

**Comment by felixfontein** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by Landrash** [comment]
+1

**Comment by acozine** [comment]
+1 there's great work there. If we want additional changes and improvements, we can open a new PR for those.

**Comment by Warkdev** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by oraNod** [comment]
+1

**Comment by lucab85** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by tima** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by samccann** [comment]
+1
